### PR TITLE
update config documentation

### LINKED ISSUE #2166: config documentation update
**Describe the bug**
I noticed there are a few places that `config --set` is referenced in the documentation. Config has recently changed and doesn't have flag like it used to. So, it would be nice to get these occurences updated from `config --set` to `config set`. I think this is a complete list of config subcommands that need to be changed.
* clear
* get
* load
* path
* remove
* set
* set_into

### PR DESCRIPTION
**Pull request by morrme**
Fixes #2166

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,11 +236,11 @@ Here we use the variable `$it` to refer to the value being piped to the external
...
 Nu has early support for configuring the shell. You can refer to the book for a list of [all supported variables](https://www.nushell.sh/book/en/configuration.html).
 
-To set one of these variables, you can use `config --set`. For example:
+To set one of these variables, you can use `config set`. For example:
 
 ```shell
-> config --set [edit_mode "vi"]
-> config --set [path $nu.path]
+> config set [edit_mode "vi"]
+> config set [path $nu.path]
```
Comment (andrasio) [inline README.md:243]: these should be `config set edit_mode "vi"` and `config set path $nu.path` respectively.

Try `help config set` and same for the others to make sure it's correct. (in this case `config set` no longer takes a table as an argument but rather positionals.

Thanks!

Comment (fdncred) [reply README.md:243]: @andrasio I personally like the tables better than just a string of parameters. I wish we'd unify our terms. It's hard to remember when to do `[some thing]` and when to do `some thing`

Comment (morrme) [reply README.md:243]: @andrasio I seem to have built the wrong version, as I still have the "old" commands.

Comment (Tauheed-Elahee) [reply README.md:243]: @morrme Could you also update the "Configuring Nu as a login shell" section of the [configuration page of the Nu book](https://www.nushell.sh/book/en/configuration.html) as well?

Comment (morrme) [reply README.md:243]: > @morrme Could you also update the "Configuring Nu as a login shell" section of the [configuration page of the Nu book](https://www.nushell.sh/book/en/configuration.html) as well?

@tauheed-elahee sure!  This one should be wrapped up by tomorrow so expect a PR there soon!

File: docs/commands/alias.md [hunk 2]
```diff
@@ -54,23 +54,23 @@ flags:
 Aliases are most useful when they are persistent. For that, add them to your startup config:
 
 ```shell
-> config --set [startup ["alias myecho [msg] { echo $msg }"]]
+> config set [startup ["alias myecho [msg] { echo $msg }"]]
```
Comment (jonathandturner) [inline docs/commands/alias.md:57]: This also needs an update to be: `config set startup ["alias myecho [msg] { echo $msg }"]`

Comment (jonathandturner) [reply docs/commands/alias.md:57]: Actually, let me see if I can just make the tweak and get this landed.

Comment (morrme) [reply docs/commands/alias.md]: :+1:

**Comment by jonathandturner** [comment]
Great, thanks again 😃

**Pull request merged**